src/Raven.Server/Documents/Handlers/AI/Agents/ConversationHandler.cs

```diff
@@ -714,8 +671,10 @@ private async Task<bool> TryHandleActionResponses(JsonOperationContext context)
             await HandleSubAgentCalls(context, subAgentsActions);
         }
 
-        if (_document.OpenActionCalls.Count > 0)
+        if (_document.OpenActionCalls.Any(x => x.Value.Type != AiAgentActionRequestType.SubAgent) || _childUserCalls.Count > 0)
```

Comment:
why not simply `_document.OpenActionCalls.Count > 0 || _childUserCalls.Count > 0`?